### ISSUE TITLE
Show application screenshots instead of icons

### ISSUE THREAD

**Issue by savchenko** [ErikReider/SwayFloatingSwitcher#5]
This one might be impossible due to Wayland limitation, but consider this, much better overview of what's going on:

![image](https://user-images.githubusercontent.com/300146/141859160-99d948ff-19ac-4a53-bf8c-331f3ccda6ad.png)

**Comment by ErikReider** [comment]
I don't think that wlroots supports capturing each application preview yet. It does support whole monitor support though
